packages/suite/src/views/wallet/transactions/components/TransactionList/components/DayHeader/index.tsx

```diff
@@ -12,13 +12,13 @@ import {
 import { parseKey } from '@wallet-utils/transactionUtils';
 import { isTestnet } from '@wallet-utils/accountUtils';
 import { Network } from '@wallet-types';
+import { SECONDARY_PANEL_HEIGHT } from '@suite-components/AppNavigation';
 
 const Wrapper = styled.div`
     display: flex;
```

Comment:
![Screenshot 2022-05-18 at 9 43 41](https://user-images.githubusercontent.com/33235762/168985149-2d42483e-7880-4508-88e7-09fd069e0e93.png)

Reply:
Fixed. Same problem in _Dropdown_ as above.